third_party/README.rst

```diff
@@ -15,15 +15,15 @@ This part of the repository contains extra dependencies required to build DALI,
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
 | `CV-CUDA <https://github.com/CVCUDA/CV-CUDA>`_                   | `0.7.0 beta <https://github.com/CVCUDA/CV-CUDA/releases/tag/v0.8.0-beta>`_                                                | `Apache License 2.0 <https://github.com/CVCUDA/CV-CUDA/blob/main/LICENSE.md>`_                                    |
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
-| `DLPack <https://github.com/dmlc/dlpack>`_                       | `0.8 <https://github.com/dmlc/dlpack/releases/tag/v0.8>`_                                                                 | `Apache License 2.0 <https://github.com/dmlc/dlpack/blob/main/LICENSE>`_                                          |
+| `DLPack <https://github.com/dmlc/dlpack>`_                       | `1.0 <https://github.com/dmlc/dlpack/releases/tag/v1.0>`_                                                                 | `Apache License 2.0 <https://github.com/dmlc/dlpack/blob/main/LICENSE>`_                                          |
 +------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------+
-| `FFTS <https://github.com/JanuszL/ffts>`_                        | `Custom fork top-of-tree (Jan 23, 2020) <https://github.com/JanuszL/ffts/tree/c9a9f61a60505751cac385ed062ce2720bdf07d4>`_ | `BSD 3-Clause License <https://github.com/JanuszL/ffts/blob/master/COPYRIGHT>`_                                   |
+| `FFTS <https://github.com/JanuszL/ffts>`_                        | `Custom fork top-of-tree (Sep 6, 2024) <https://github.com/JanuszL/ffts/tree/95489ebcd6fc136c6a76f50f57d43e6072e2bd38>`_  | `BSD 3-Clause License <https://github.com/JanuszL/ffts/blob/master/COPYRIGHT>`_                                   |
```

Comment:
Good idea. I forgot that I updated my fork to support more recent gcc compiler available ion manylinux_2_28 image - due to pending https://github.com/NVIDIA/DALI/pull/5608.